tests/integration/test_keeper_force_recovery_single_node/configs/enable_keeper3.xml

```diff
@@ -1,5 +1,6 @@
 <clickhouse>
     <keeper_server>
+        <use_cluster>false</use_cluster>
```

Comment:
Ok. It means, "don't use this config for the client". But I don't understand why this change was needed.

Reply:
To reduce the number of logs we create on Keeper. Recovery has some troubles if there are too many configuration changes and logs to sync on new nodes.
This is hard to solve with current approach but a better and easier way of recovery can now be defined using new `reconfig` (https://github.com/ClickHouse/ClickHouse/pull/49450).